### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by vladimyr**
This PR fixes build process to produce es5 artifacts using babel.

### CODE REVIEW

**Comment by vladimyr** [comment]
@Dafrok Please publish new version when you merge this one because current build is broken (es6 code is being shipped instead of es5). ⚠️

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+module.exports = {
+  presets: ['@babel/preset-env']
+}
```
Comment (Dafrok) [inline babel.config.js]: Are there something different from babelrc file?

File: package.json [hunk 2]
```diff
@@ -7,47 +7,28 @@
...
-    "vue",
-    "hotkey",
-    "hotkeys",
-    "directive",
-    "shortcut",
-    "shortcuts"
-  ],
-  "license": "MIT",
   "scripts": {
+    "build": "vue-cli-service build --target lib --name v-hotkey ./src/index.js",
```
Comment (Dafrok) [inline package.json]: I found these are just changes in order. What's the purpose of doing this?

File: babel.config.js (hunk 1 above)
Comment (vladimyr) [reply babel.config.js]: It was created when i ran:
```bash
vue add babel # vue is @vue/cli
```
inside the project. It is different in sense its a future proof solution that forces all dependencies to be transpiled together with your code: https://babeljs.io/docs/en/configuration#whats-your-use-case

File: package.json (hunk 2 above)
Comment (vladimyr) [reply package.json]: Rearrengement happened as side effect running vue cli inside the project but important change here, somehow hidden from plain sight 🙁 , is that `@vue/cli-plugin-babel` is being added to dev dependencies. You can safely ignore all other ordering changes.

Comment (Dafrok) [reply package.json]: That's OK

**Comment by Dafrok** [comment]
https://github.com/Dafrok/v-hotkey/commit/9ed993a0ab6100ee72a7b767baabf7d9e0239d58

Package has been updated.

**Pull request merged**